cmake/CMakeLists.txt

```diff
@@ -222,10 +222,10 @@ if(onnxruntime_MINIMAL_BUILD)
   else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffunction-sections -fdata-sections")
     if (CMAKE_HOST_SYSTEM MATCHES "Darwin")
-      add_link_options(-Wl,-dead_strip)
+      add_link_options(-Wl, -dead_strip)
```

Comment:
>  [](start = 27, length = 1)

I have to add a space here, otherwise build on Mac will fail, Android CI is running on Mac now